web/src/components/profile/ProfileBusinessStructure.tsx

```diff
@@ -0,0 +1,72 @@
+import { ArrowTooltip } from "@/components/ArrowTooltip";
+import { Content } from "@/components/Content";
+import { Icon } from "@/components/njwds/Icon";
+import { FieldLabelProfile } from "@/components/onboarding/FieldLabelProfile";
+import { ConfigType } from "@/contexts/configContext";
+import { ProfileDataContext } from "@/contexts/profileDataContext";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useRoadmap } from "@/lib/data-hooks/useRoadmap";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { getProfileConfig } from "@/lib/domain-logic/getProfileConfig";
+import { isGetFilingResponseSuccussful } from "@/lib/domain-logic/isGetFilingResponseSuccussful";
+import { getTaskFromRoadmap } from "@/lib/utils/roadmap-helpers";
+import { LookupLegalStructureById } from "@businessnjgovnavigator/shared/legalStructure";
+import { ReactElement, useContext, useMemo } from "react";
+
+export const ProfileBusinessStructure = (): ReactElement => {
+  const { state } = useContext(ProfileDataContext);
+  const { Config } = useConfig();
+  const { roadmap } = useRoadmap();
+  const { updateQueue } = useUserData();
+  const isFormed = isGetFilingResponseSuccussful(updateQueue?.current());
+
+  const contentFromConfig: ConfigType["profileDefaults"]["fields"]["legalStructureId"]["default"] =
+    getProfileConfig({
+      config: Config,
+      persona: state.flow,
+      fieldName: "legalStructureId",
+    });
+
+  const businessStructureTaskUrl = useMemo(() => {
+    const urlSlug = getTaskFromRoadmap(roadmap, "business-structure")?.urlSlug ?? "";
+    return `/tasks/${urlSlug}`;
+  }, [roadmap]);
+
+  return (
+    <div data-testid="business-structure">
+      <div className="flex flex-row">
+        <FieldLabelProfile fieldName="legalStructureId" />
+        {isFormed ? (
+          <div className="margin-left-2">
+            <ArrowTooltip
+              title={Config.profileDefaults.lockedFieldTooltipText}
+              data-testid="business-structure-tooltip"
+            >
+              <div className="fdr fac font-body-lg">
+                <Icon>help_outline</Icon>
```

Comment:
This is copy-pasted from FieldLabelProfile - could we make use of the `locked` prop on that instead of rewriting it here?

ie would this work?

```
<FieldLabelProfile fieldName="legalStructureId" locked={isFormed}/>
{!isFormed && <a
            className="margin-left-2"
            href={businessStructureTaskUrl}
            data-testid={"business-structure-task-link"}
          >
            {state.profileData.legalStructureId ? contentFromConfig.editText : contentFromConfig.addText}
          </a>
}
```